pkgs/tools/admin/qovery-cli/default.nix

```diff
@@ -16,6 +19,20 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-4TY7/prMbvw5zVPJRoMLg7Omrxvh1HPGsdz1wqPn4uU=";
 
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion --cmd ${pname} \
+      --bash <($out/bin/${pname} completion bash) \
+      --fish <($out/bin/${pname} completion fish) \
+      --zsh <($out/bin/${pname} completion zsh)
```

Comment:
Please do not substitute pname wherever possible. Especially in pythonPackages where pname might actually change when normalising the package name, it causes more maintenance time than it saves.